File: app/src/main/java/com/mutant/godutch/ViewHolderFriend.java
package com.mutant.godutch;

import android.support.v7.widget.AppCompatImageView;
import android.support.v7.widget.AppCompatTextView;
import android.support.v7.widget.RecyclerView;
import android.view.View;

/**
 * Created by evanfang102 on 2017/3/30.
 */

public class ViewHolderFriend extends RecyclerView.ViewHolder {

    public AppCompatImageView mImageViewProPic;
    public AppCompatTextView mTextViewName;
    public AppCompatTextView mTextViewInvitationState;

    public ViewHolderFriend(View itemView) {
        super(itemView);
        findViews(itemView);
    }

    private void findViews(View itemView) {
        mImageViewProPic = (AppCompatImageView) itemView.findViewById(R.id.imageView_pro_pic);
        mTextViewName = (AppCompatTextView) itemView.findViewById(R.id.textView_name);
        mTextViewInvitationState = (AppCompatTextView) itemView.findViewById(R.id.textView_invitation_state);
    }

}

File: app/src/main/java/com/mutant/godutch/ViewHolderFriend.kt
package com.mutant.godutch

import android.support.v7.widget.AppCompatImageView
import android.support.v7.widget.AppCompatTextView
import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView
import kotlinx.android.synthetic.main.card_view_item_friend.view.*

/**
 * Created by evanfang102 on 2017/3/30.
 */

class ViewHolderFriend(itemView: View) : RecyclerView.ViewHolder(itemView) {

    var mImageViewPhotoUrl: AppCompatImageView = itemView.imageView_photo_url
    var mTextViewName: AppCompatTextView = itemView.textView_name
    var mTextViewInvitationState: AppCompatTextView = itemView.textView_invitation_state
    var mTextViewDebt: TextView = itemView.textView_debt

}
